### PR TITLE
Multiple email templates option upon rejecton

### PR DESCRIPTION
**Pull request by rajatsajwan**
Targets #2687

### Description
Creating Multiple email templates option upon rejection Template with

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: resources/js/app.js [hunk 1]
```diff
@@ -2025,6 +2025,39 @@ $(".opt").on("click", function() {
 	});
 });
 
+$(".rejectMailOpt").on("click",function(){
```
Comment (P4NK4J) [inline resources/js/app.js:2028]: If this is a classname, why it is camelcased ?

File: resources/js/app.js [hunk 2]
```diff
@@ -2025,6 +2025,39 @@ $(".opt").on("click", function() {
...
+	let formData = {
+		setting_key_subject: $(this).data("key-subject"),
+		setting_key_body: $(this).data("key-body"),
+		applicant_name: $("#applicantName").text(),
+		job_title: $("#jobTitle").text(),
+	};
+
+	var originUrl = window.location.origin;
```
Comment (P4NK4J) [inline resources/js/app.js:2036]: can we define this variable as a global one ?

File: resources/views/hr/application/rejection-modal.blade.php [hunk 3]
```diff
@@ -69,15 +69,56 @@
...
+                            <li class="nav-item"><a data-toggle="tab" href="#codeTrekProposition" class="nav-link rejectMailOpt" data-key-subject='codetrek_proposition_subject' data-key-body='codetrek_proposition_body'>CodeTrek Proposition</a></li>
+                        </ul>
+                        <div class="tab-content">
+                            <div class="tab-pane" id="generalCommunication">
+                                <div class="card-body">
+                                    <div class="form-row">
+                                        <div class="col-md-12">
+                                            <div class="form-group">
+                                                <label>Subject</label>
+                                                <input name="subject_general_communication" type="text" class="form-control option-subject" id="general_communicationSubject">
```
Comment (P4NK4J) [inline resources/views/hr/application/rejection-modal.blade.php:85]: What is the naming convention you are using in id ? Is it camelcase or snakecase ? Please avoid these silly mistakes

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Left some comments.

File: resources/views/hr/application/rejection-modal.blade.php (hunk 3 above)
Comment (rajatsajwan) [reply resources/views/hr/application/rejection-modal.blade.php]: i change it into snakecase

**Comment by P4NK4J** [comment]
> Screenshot (63) sir this is running fine in my system.

If it's working on your local, it doesn't mean it would work on the production server as well. It's working on your system because you started with a. fresh database. Try importing the UAT one and see with existing database what's happening

**Comment by rajatsajwan** [comment]
image

this is result in general communication

**Comment by rajatsajwan** [comment]
image

this is  codetrack

**Comment by rajatsajwan** [comment]
image

result of changes.

**Comment by rajatsajwan** [comment]
image

result of changes.

**Comment by rajatsajwan** [comment]
image

result of changes.

**Comment by rajatsajwan** [comment]
Screenshot (65)

this is result of changes

**Pull request closed without merging**